src/ContentBundle/Resources/views/Block/Content/login.html.twig

```diff
@@ -0,0 +1,77 @@
+
+	                    {% if error is defined and error is not null %}
+	                        <div class="alert alert-danger alert-inline" style="margin-bottom:15px;">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
+	                    {% endif %}
+
+	                    <form action="{{ path("fos_user_security_check") }}" method="post" role="form" class="form-horizontal">
+	                        <input type="hidden" name="_csrf_token" value="{{ csrf_token }}" />
+	                        <div class="form-group">
+	                            <label for="username" class="col-sm-2 control-label">
+	                                {{ 'security.login.username'|trans }}
+	                            </label>
+	                            <div class="col-sm-10">
+	                                <input type="text" id="username" name="_username" value="{{ last_username }}" required="required" class="form-control" />
+	                            </div>
+	                        </div>
+
+	                        <div class="form-group">
+	                            <label for="password" class="col-sm-2 control-label">
+	                                {{ 'security.login.password'|trans }}
+	                            </label>
+	                            <div class="col-sm-10">
+	                                <input type="password" id="password" name="_password" required="required" class="form-control" />
+	                            </div>
+	                        </div>
+
+	                        {% if google_captcha_site_key is defined %}
+	                        <div class="form-group">
+	                            <div class="col-sm-offset-3 col-sm-9">
+	                                <div class="g-recaptcha" data-sitekey="{{ google_captcha_site_key }}"></div>
+	                            </div>
+	                        </div>
+	                        {% endif %}
+
+	                        <div class="form-group">
+	                            <div class="col-sm-offset-2 col-sm-10">
+	                                <div class="checkbox">
+	                                    <label for="remember_me">
+	                                        <input type="checkbox" id="remember_me" name="_remember_me" checked="checked" />{{ 'security.login.remember_me'|trans }}
+	                                    </label>
+	                                    {% if registrationUrl|default or block.registrationContentItem|default %}
+			                            	{% if block.registrationContentItem|default %}	
+												{% set registrationUrl = path('_content', {'slug': block.registrationContentItem.slug }) %}
+											{% else %}
+												{% if block.properties.registrationUrl|default %} {% set registrationUrl = block.properties.registrationUrl %} {% endif %}
+											{% endif %}
+				                            	 | <a href="{{ registrationUrl }}">Register</a>
+				                        {% endif %}
```

Comment:
Can you please format this file properly? If it already looks fine on your system, could you make sure you're using 4-spaces instead of tabs for indentation?